web3_proxy/src/rpcs/request.rs

```diff
@@ -299,87 +308,127 @@ impl OpenRequestHandle {
                 if let Some(msg) = msg {
                     if msg.starts_with("execution reverted") {
                         trace!("revert from {}", self.rpc);
-                        ResponseTypes::Revert
+                        ResponseErrorType::Revert
                     } else if msg.contains("limit") || msg.contains("request") {
                         trace!("rate limit from {}", self.rpc);
-                        ResponseTypes::RateLimit
+                        ResponseErrorType::RateLimit
                     } else {
-                        ResponseTypes::Ok
+                        ResponseErrorType::Error
                     }
                 } else {
-                    ResponseTypes::Ok
+                    ResponseErrorType::Error
                 }
             } else {
-                ResponseTypes::Ok
+                ResponseErrorType::Error
```

Comment:
this one isn't an error. this is an okay response

Reply:
Should be taken over from main as well now